src/Buckle/Buckle.Tests/CodeAnalysis/Syntax/LexerTests.cs

```diff
@@ -324,27 +324,27 @@ public class LexerTests {
         if (t1Type == SyntaxType.GREATER_THAN_GREATER_THAN_TOKEN && t2Type ==
             SyntaxType.GREATER_THAN_GREATER_THAN_EQUALS_TOKEN)
             return true;
-        if (t1Type == SyntaxType.GREATER_THAN_GREATER_THAN_TOKEN && t2Type ==
-            SyntaxType.GREATER_THAN_EQUALS_TOKEN)
+        if (t1Type == SyntaxType.GREATER_THAN_GREATER_THAN_TOKEN && t2Type == SyntaxType.GREATER_THAN_EQUALS_TOKEN)
             return true;
-        if (t1Type == SyntaxType.GREATER_THAN_GREATER_THAN_TOKEN && t2Type ==
-            SyntaxType.GREATER_THAN_TOKEN)
+        if (t1Type == SyntaxType.GREATER_THAN_GREATER_THAN_TOKEN && t2Type == SyntaxType.GREATER_THAN_TOKEN)
             return true;
         if (t1Type == SyntaxType.GREATER_THAN_GREATER_THAN_TOKEN && t2Type ==
             SyntaxType.GREATER_THAN_GREATER_THAN_TOKEN)
             return true;
-        if (t1Type == SyntaxType.PERCENT_TOKEN && t2Type ==
-            SyntaxType.EQUALS_EQUALS_TOKEN)
+        if (t1Type == SyntaxType.PERCENT_TOKEN && t2Type == SyntaxType.EQUALS_EQUALS_TOKEN)
             return true;
-        if (t1Type == SyntaxType.PERCENT_TOKEN && t2Type ==
-            SyntaxType.EQUALS_TOKEN)
+        if (t1Type == SyntaxType.PERCENT_TOKEN && t2Type == SyntaxType.EQUALS_TOKEN)
             return true;
         if (t1Type == SyntaxType.GREATER_THAN_GREATER_THAN_GREATER_THAN_TOKEN && t2Type ==
             SyntaxType.EQUALS_EQUALS_TOKEN)
             return true;
         if (t1Type == SyntaxType.GREATER_THAN_GREATER_THAN_GREATER_THAN_TOKEN && t2Type ==
             SyntaxType.EQUALS_TOKEN)
             return true;
+        if (t1Type == SyntaxType.QUESTION_QUESTION_TOKEN && t2Type == SyntaxType.EQUALS_TOKEN)
+            return true;
+        if (t1Type == SyntaxType.QUESTION_QUESTION_TOKEN && t2Type == SyntaxType.EQUALS_EQUALS_TOKEN)
+            return true;
```

Comment:
This is actually a bug fix from the null-coalescing operator. We forgot to add these if checks to make sure the lexer tests don't break.